### ISSUE TITLE
Wrong error on `super().__init__` in metaclass

### ISSUE THREAD

**Issue by JelleZijlstra** [microsoft/pyright#2872]
```python
class Metaclass(type):
    def __init__(self, name, bases, attrs):
        super().__init__(name, bases, attrs)  # pyright: Expected 1 positional argument
```
This code runs fine. Looking at https://github.com/python/typeshed/blob/master/stdlib/builtins.pyi#L151, I suspect it's picking up the wrong overload.

**Comment by erictraut** [comment]
Thanks for the bug report. This will be addressed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.210, which I just published. It will also be included in the next release of pylance.
